src/main/java/Duke.java

```diff
@@ -1,10 +1,230 @@
+
+    private static void mark(int n, ArrayList<Task> tasks) {
+        if ((n > 0) && (tasks.get(n - 1) != null)) {
+            tasks.get(n - 1).setDone();
+            System.out.println("Nice! I've marked this task done:");
+            System.out.println("[" + tasks.get(n - 1).getStatusIcon() + "] "
+                    + tasks.get(n - 1).getDescription() + "\n");
+        } else {
+            System.out.println("You don't have that task!\n");
+        }
+    }
+
+    private static void unmark(int n, ArrayList<Task> tasks) {
+        if ((n > 0) && (tasks.get(n - 1) != null)) {
+            tasks.get(n - 1).setUndone();
+            System.out.println("OK, I've marked this task as not done yet:");
+            System.out.println("[" + tasks.get(n - 1).getStatusIcon() + "] "
+                    + tasks.get(n - 1).getDescription() + "\n");
+        } else {
+            System.out.println("You don't have that task!\n");
+        }
+    }
+
+    private static void delete(int n, ArrayList<Task> tasks) {
+        if ((n > 0) && (tasks.get(n - 1) != null)) {
+            Task removed = tasks.remove(n - 1);
+            System.out.println("Noted. I've removed this task:");
+            System.out.println(removed);
+            System.out.println("Now you have " + tasks.size() + " tasks in the list.\n");
+        } else {
+            System.out.println("You don't have that task!\n");
+        }
+    }
+
+    private static void createToDo(String task, ArrayList<Task> tasks) {
+        ToDo todo = new ToDo(task);
+        tasks.add(todo);
+        System.out.println("Got it. I've added this task:");
+        System.out.println(todo);
+        System.out.println("Now you have " + tasks.size() + " tasks in the list.\n");
+    }
+
+    private static void createDeadline(String description, String by, ArrayList<Task> tasks) {
+        Deadline deadline = new Deadline(description, by);
+        tasks.add(deadline);
+        System.out.println("Got it. I've added this task:");
+        System.out.println(deadline);
+        System.out.println("Now you have " + tasks.size() + " tasks in the list.\n");
+    }
+
+    private static void createEvent(String description, String from, String to, ArrayList<Task> tasks) {
+        Event event = new Event(description, from, to);
+        tasks.add(event);
+        System.out.println("Got it. I've added this task:");
+        System.out.println(event);
+        System.out.println("Now you have " + tasks.size() + " tasks in the list.\n");
+    }
+
+    private static void storeTasks(ArrayList<Task> tasks) {
+        for (Task task : tasks) {
+            try {
+                task.writeToFile(storedTasksPath);
+            } catch (IOException e) {
+                System.out.println("Something went wrong: " + e.getMessage());
+            }
+        }
+    }
+
+    private static void loadTasks(Scanner sc, ArrayList<Task> tasks) {
+        while (sc.hasNext()) {
+            String current = sc.nextLine();
```

Comment:
Perhaps a more intuitive variable name here? Maybe currentLine instead of current?